### ISSUE TITLE
[security] Rancher server image comes with known vulnerabilities

### ISSUE THREAD

**Issue by IPv4v6** [rancher/rancher#8676]
**Rancher Versions:**
Server: 1.6.0

**The image rancher/server:v1.6.0 (Image ID f0ed20ddb71d) from Docker hub contains vulnerabilities.**

The package libc6 2.19-0ubuntu6.7 is installed, whereas 2.19-0ubuntu6.11 is available:

```
# apt-cache policy libc6
libc6:
  Installed: 2.19-0ubuntu6.7
  Candidate: 2.19-0ubuntu6.11
  Version table:
     2.19-0ubuntu6.11 0
        500 http://archive.ubuntu.com/ubuntu/ trusty-updates/main amd64 Packages
        500 http://archive.ubuntu.com/ubuntu/ trusty-security/main amd64 Packages
 *** 2.19-0ubuntu6.7 0
        100 /var/lib/dpkg/status
     2.19-0ubuntu6 0
        500 http://archive.ubuntu.com/ubuntu/ trusty/main amd64 Packages
```

Check Ubuntu Security Notice USN-3239-1 for vulnerability details and CVE numbers.
https://www.ubuntu.com/usn/usn-3239-1/

**Comment by IPv4v6** [comment]
Can you please fix your Rancher server image?

According to Docker hub statistics the server image is very popular and I suppose a lot of people use it in production.

**Comment by loganhz** [comment]
Fixed in v1.6.22
